Pcap++/CMakeLists.txt

```diff
@@ -4,20 +4,20 @@ add_library(
   $<$<BOOL:${PCAPPP_USE_DPDK}>:src/DpdkDeviceList.cpp>
   $<$<BOOL:${PCAPPP_USE_DPDK}>:src/KniDevice.cpp>
   $<$<BOOL:${PCAPPP_USE_DPDK}>:src/KniDeviceList.cpp>
-  $<$<BOOL:${PCAPPP_TARGET_PLATFORM_LINUX}>:src/LinuxNicInformationSocket.cpp>
+  $<$<BOOL:${UNIX}>:src/LinuxNicInformationSocket.cpp>
```

Comment:
`UNIX` can be any [unix-like system](https://gitlab.kitware.com/cmake/community/-/wikis/doc/tutorials/How-To-Write-Platform-Checks#platform-variables) including Apple OS X and CygWin.

Maybe we want to use [`CMAKE_SYSTEM_NAME`](https://gitlab.kitware.com/cmake/community/-/wikis/doc/tutorials/How-To-Write-Platform-Checks#cmake-system) instead?

Reply:
I have pushed a fix